infra/hooks/postprovision.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# Load the azd environment variables
+DIR=$(dirname "$(realpath "$0")")
+"$DIR/load_azd_env.sh"
```

Comment:
I don't think we need to load the env in hooks since it is already loaded and we now have azd hook run